config/routes.rb

```diff
@@ -2,5 +2,8 @@
   root 'home#index'
   resources :cards
   resources :users
+  resources :user_sessions
+  get 'login' => 'user_sessions#new', as: :login
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.